fenix/app/src/main/java/org/mozilla/fenix/shopping/ReviewQualityCheckFeature.kt

```diff
@@ -42,9 +49,20 @@ class ReviewQualityCheckFeature(
                 .distinctUntilChanged()
                 .collect(onAvailabilityChange)
         }
+
+        appStoreScope = appStore.flowScoped { flow ->
+            flow.mapNotNull { it.shoppingSheetExpanded }
+                .distinctUntilChanged()
+                .drop(1) // Needed to ignore the initial emission from the Store setup
```

Comment:
Let's set null as default value for expanded property in `AppState` and remove the `drop`. This can lead to issues, we don't always know if flow will begin emitting from initial state.